### PR TITLE
Feat: Added product listing page

### PR DESCRIPTION
**Pull request by bmendhe23**
##Description
Feat:
1. Added product listing page to the store website.

##Screenshots
ss

##Platforms tested
[✔️] Desktop
[❌] Tablet
[❌] Mobile

### CODE REVIEW

File: Product/product.html [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+            </section>
+            <section class="searchbar">
+                <div class="input-icon">
+                    <i class="material-icons">search</i>
+                    <input  class="input" type="text" placeholder="Search">
+                </div>
+            </section>
+            <section class="nav-links">
+                <button class="btn">Login</button>
+                <div class="badge">
```
Comment (sashaboi) [inline Product/product.html:31]: Might need hover cursor pointer and some sort of bg color animation ? could help UX

File: Product/product.html [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+                    <div class="number dnd">0</div>
+                </div>
+                <div class="badge">
+                    <i class="material-icons-outlined">shopping_cart</i>
+                    <div class="number dnd">0</div>
+                </div>
+            </section>
+        </nav>
+    </header>
+    <div class="wrapper">
```
Comment (sashaboi) [inline Product/product.html:42]: These categories might need the pointer treatment as well ? might help UX

File: Product/product.html [hunk 3]
```diff
@@ -0,0 +1,198 @@
...
+                </div>
+                <div class="radio">
+                    <label for="high-to-low">
+                        <input name="price-sort" type="radio">
+                        Price - High to Low
+                    </label>
+                </div>
+            </div>
+        </aside>
+        <main class="products-container">
```
Comment (sashaboi) [inline Product/product.html:115]: The cards as shown in PRD are vertical , are we sure the horizontal cards will be ok? hope they are , do confirm with TAs!

**Review by sashaboi — APPROVED** [review]
Everything looks amazing and clean , good to go !
I have added a few comments do go through!
Onkar Deshpande

File: Product/product.html (hunk 1 above)
Comment (bmendhe23) [reply Product/product.html:31]: sure will do that @sashaboi

**Review by harshitpaliwal95 — COMMENTED** [review]
looks good

**Pull request merged**